### ISSUE TITLE
Error on importing MySQLdb right after a successful install on Windows 7, Python 3.8 64-bit

### ISSUE THREAD

**Issue by ShinyExample** [PyMySQL/mysqlclient-python#402]
Hi!

I think this is an issue in PyMySQL:
I get an error on importing MySQLdb right after a successful install:

```
>>> import MySQLdb
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\...\python38-64-bit\lib\site-packages\MySQLdb\__init__.py", line 21, in <module>
    raise ImportError("this is MySQLdb version %s, but _mysql is version %r" %
ImportError: this is MySQLdb version (1, 4, 5, 'final', 0), but _mysql is version (1, 4, 4, 'final', 0)
```

HOW INSTALLED IT ON WINDOWS 7, 64-bit on Python 3.8 64-bit:

I installed succesfully Python using:
python-3.8.0-amd64.exe

Then I successfully did:

```
> python -m pip install --upgrade pip
```

Then I successfully did:

```
> pip install mysqlclient
Collecting mysqlclient
  Using cached https://files.pythonhosted.org/packages/af/f4/0121c2d45e44ddb70cf3cbd4be0b22e0d90cf56d26d1addd17be09a33ba1/mysqlclient-1.4.5-cp38-cp38-win_amd64.whl
Installing collected packages: mysqlclient
Successfully installed mysqlclient-1.4.5
```

**Comment by methane** [comment]
I confirmed the binary wheel for Python 3.8 was broken and I removed it from PyPI.
Since PyPI doesn't allow adding a new wheel for the same version, please wait until the new release.

**Comment by ShinyExample** [comment]
Oh dear, I actually need to use right now. Allright, I'll downgrade to Python 3.7 then so I can use the 1.4.4.

**Comment by methane** [comment]
Released v1.4.6
